tests/fixtures/testModel.ts

```diff
@@ -34,6 +35,7 @@ export interface TestModel extends Model {
    */
   numberValue: number;
   numberArray: number[];
+  readonly numberArrayReadonly: readonly number[];
```

Comment:
This could have validation layer implications, which is why a previous attempt ultimately didn't make the cut.
It may be fine to accept readonly (in TS lang) as settable properties, but I lean towards `readonly` (TS) should be `readOnly` (OAS).
Reference: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#fixed-fields-20